src/master-playlist-controller.js

```diff
@@ -161,6 +162,10 @@ export class MasterPlaylistController extends videojs.EventTarget {
     this.useCueTags_ = useCueTags;
     this.blacklistDuration = blacklistDuration;
     this.enableLowInitialPlaylist = enableLowInitialPlaylist;
+    this.maxBlacklistedPlaylistReloads = maxBlacklistedPlaylistReloads;
+
+    this.blacklistedPlaylistReloads = 0;
```

Comment:
I think this should be suffixed with a `_` and renamed similarly to how we end up renaming `maxBlacklistedPlaylistReloads`

Reply:
Renamed to `finalRenditionRetries_`